rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,151 @@
+- Organizational Structure, that enacts necesary degrees of _Division of Labor_ 
+  and _Coordination_.
+- Organizational Culture, that helps to overcome the complexity bias. ([KISS](https://en.wikipedia.org/wiki/KISS_principle))
+
+Analysis inspired by [New St.Gallen Managament Model].
+
+[New St.Gallen Managament Model]: https://cio-wiki.org/wiki/The_New_St._Gallen_Management_Model
+
+# Detailed design
+[design]: #detailed-design
+
+A repository shall exist, named `NixOS/community`, which acts as central hub
+for coordinating the community dynamics.
+
+It shall comprehend, consolidate and showcase aspects of:
+
+- Communication
+  - Channels
+  - Modes & Policies
+- Governance
+  - Processes
+  - Structure
+- Onboarding
+  - Code Contributions
+  - Community Engagement
+
+It shall omit any technical subject about nix or code in general, other than
+repository tooling.
+
+To implement, a detailed review of https://github.com/kubernetes/community
+shall be conducted as joint effort between the interested parties and as 
+an Annex I to this RFC.
+
+Based on this review, a clean repository shall be enacted implementing
+such aspects considered worth porting to our community.
+
+As a general guideline for this review, the vast success and impressive
+dynamics of the kubernetes ecosystem shall be acknowledged not only as a
+proof of concept, but as a very sucessfull and inspiring [viable system].
+
+[viable system]: https://en.wikipedia.org/wiki/Viable_system_theory
+
+# Brainstorm
+[brainstorm]: #brainstorm
+
+This section exposes loose ideas that have sparked interest of commenters
+in their order of submission. They shall be considered _optional_ material
+during implementation, whereas implementors shall have the liberty to
+deliver a consistent experience as they see fit.
+
+- [Community Event Calender](https://github.com/kubernetes/community/tree/master/events)
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+## As a newcomer
+As a newcomer, I'd be able to quickly find my way around, and start beeing
+useful before beeing turned down by inconsistent and spread-out onboarding.
+
+## As a veteran
+As a veteran, I'd be able to promote some ideas, that I care about,
+towards completion while suffering from less friction and overly complex
+coordination. I can leverage my experience and skills more effectively
+for the benefit of the community.
+
+## As a mentor
+As a mentor, I can _still_ \* maintain those high quality direct human
+interactions, while having full support of a _"curriculum-like"_ resource.
+
+\* _still_, as an aside, because nix community is known for it's great
+mentorship spirit.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+There _can_ be a better solution to the achieve _optimum levels of empowerment_
+for our community. We _could just **wait**_ for them to unveil themselves.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Benevolent Dictatorship
```

Comment:
Link to https://discourse.nixos.org/t/what-would-you-do-if-you-were-the-bdfl-of-nix/6949 would be nice :).